package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest",
     "lint": "eslint src/ --fix",
-    "build": "babel src --out-dir lib --ignore **/*.test.js",
+    "build": "babel src --out-dir lib --ignore *.test.js",
```

Comment:
The previous pattern was not matching because all of the test files are at the root of the src folder. Now the tests are not being built into lib 💃